inspire_utils/name.py

```diff
@@ -76,13 +79,30 @@ def __init__(self, name, constants=None):
             constants = ParsedName.constants
 
         self._parsed_name = HumanName(name, constants=constants)
+        self._parsed_name.capitalize()
 
         # In the case of one name part, i.e. only lastname, name parser adds the name part in one of the first, middle,
         # suffix fields (depending on input). For our use-case if a user types a name, usually would be a lastname.
-        if len(name.split()) == 1:
-            self._parsed_name.last = self._parsed_name.first + self._parsed_name.middle + self._parsed_name.suffix
+        if not self._has_more_than_one_parts(name):
+            self._parsed_name.last = ''.join([
```

Comment:
you should join with `u''`, and you can drop the `[`...`]` inside the `join` as it can take an iterable.